sintr_analysis_server_example/lib/stacktrace_model.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library sintr_analysis_server_example.stacktrace_model;
+
+import 'package:nectar/clusterers.dart' as cluster;
+import 'package:nectar/distance_measures.dart' as distances;
+
+class StackTrace {
+  List<String> locations;
+  String fullMessage;
+  int count;
+
+  StackTrace(this.locations, [this.fullMessage, this.count]);
```

Comment:
Is ```count``` really optional? It is assumed to be non-null in the ```main``` method below. Should it be non-optional? Should it have a default value of 1?